person_class.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals, :parent_permission
```

Comment:
- [ ] I suggest that you make `rentals` `attr_accessor` as well to be able to access and modify it